tests/benchmark/snappy/dynamic/spark/PerfRun.conf

```diff
@@ -0,0 +1,74 @@
+#Do not change the way of queries, sparkProperties, sparkSqlProperties. Just change the values inside strings
```

Comment:
Before running the perf test, we create a PerfRun.conf from the PerfRun.conf.template, so this file can be deleted or renamed to PerfRun.conf.template